src/main/java/lotto/domain/winning/WinningStatistics.java

```diff
@@ -0,0 +1,58 @@
+package lotto.domain.winning;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.EnumMap;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import lotto.domain.Lottos;
+import lotto.domain.Money;
+
+public final class WinningStatistics {
+    private final Map<WinningResult, Integer> statistics;
+
+    private WinningStatistics(Map<WinningResult, Integer> statistics) {
+        this.statistics = new EnumMap<>(statistics);
+    }
+
+    public static WinningStatistics of(WinningNumbers winningNumbers, Lottos lottos) {
+        Map<WinningResult, Integer> statistics = initStatistics();
+
+        lottos.getWinningResults(winningNumbers)
+                .forEach(result -> statistics.put(result, statistics.get(result) + 1));
```

Comment:
```suggestion
                .forEach(result -> statistics.merge(result, 1, Integer::sum));
```
으로 리팩토링 해볼 수 있을 것 같아요!

Reply:
오호 머지를 시킬수 있군요! 좋은 정보 감사합니다!!!